src/main/java/com/solvd/review/kafka/config/KafkaConsumerConfig.java

```diff
@@ -0,0 +1,45 @@
+package com.solvd.review.kafka.config;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.common.serialization.LongDeserializer;
+import org.apache.kafka.common.serialization.StringDeserializer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import reactor.kafka.receiver.KafkaReceiver;
+import reactor.kafka.receiver.ReceiverOptions;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+@Slf4j
+@Configuration
+@RequiredArgsConstructor
+public class KafkaConsumerConfig {
+
+    private static final String BOOTSTRAP_SERVERS = "localhost:9092";
```

Comment:
the host is hardcoded, moreover, it is a localhost. So far, we can't run this app on our env